### DISCUSSION TITLE
Help with "Select" component in Spartan CLI and Updating to Latest Version

### DISCUSSION THREAD

**Discussion by wassimbenr** [spartan-ng/spartan#169] [Q&A] [answered]
Hello!
Firstly, to express my appreciation for the project. I'm also interested in potentially contributing to the project in the future.

Before I raise any formal issues, I wanted to seek some guidance on a couple of points related to the CLI:
- "Select All" functionality not working as expected: When using "Select All", it errors for installed components. It should ignore them and only install new ones. I'd appreciate any insights on this behavior.
- Latest "Select" component missing: I can't install the latest "Select" component from the CLI. when running the instalation command it shows the main CLI installation menu. Any suggestions on how to proceed with installing it?

Thank you for your time and assistance.

**Comment by goetzrobin** [comment, marked as answer]
Can you open a ticket for the Select All functionality.

Select should now be available with the newest alpha and ready to rock!
